extensions/most-used-content/get_usage.R

```diff
@@ -0,0 +1,60 @@
+library(connectapi)
+
+# This file contains functions that ultimately will more likely be part of
+# connectapi. As such, I'm not using dplyr or pipes here.
```

Comment:
Be careful with this: constraining yourself today might make putting this into connectapi easier, but that only counts for the code we do put into conenctapi. So if prototyping and getting things working is easier with dplyr/pipes/etc. do that here now and we can decide to refactor that later.

Reply:
Fair! It took only a minute or two to switch from using pipes etc. to laying it out like this, but if it became a limiting factor I'd discard it :)